common/src/sql/dbinit.sql

```diff
@@ -128,6 +128,46 @@ CREATE INDEX ON omicron.public.service (
     sled_id
 );
 
+CREATE TYPE omicron.public.physical_disk_kind AS ENUM (
+  'm2',
+  'u2'
+);
+
+-- A physical disk which exists inside the rack.
+CREATE TABLE omicron.public.physical_disk (
```

Comment:
I assume aspects like the state of the disk and its position will just be added later when we can get that context?

Reply:
Yeah. I didn't add it now because I was pretty explicitly instructed that "slot # != actual physical mapping" But yeah, that would go here if I had it.